packages/cyberstorm/src/components/Dialog/Dialog.tsx

```diff
@@ -8,64 +8,32 @@ import { faXmarkLarge } from "@fortawesome/pro-solid-svg-icons";
 import { Tooltip } from "../..";
 import { classnames } from "../../utils/utils";
 
-type DialogProps = {
-  content?: ReactNode;
+export type DialogProps = {
+  children?: ReactNode;
```

Comment:
Could use `interface DialogProps extends PropsWithChildren`

Reply:
No changes for this one either, what does the :+1: reaction signify here? 😅

Reply:
That I've fixed it, but it seems that I've walked back the type changes at somepoint after that. Idk, I might do the same mistakes again, while fixing a later comment.

But the 👍 always means that I've fixed it, when I react with that.